WordPress/WordPressTest/Dashboard/Dynamic Cards/DashboardDynamicCardAnalyticsEventTests.swift

```diff
@@ -0,0 +1,24 @@
+import Nimble
+@testable import WordPress
+import XCTest
+
+final class DashboardDynamicCardAnalyticsEventTests: XCTestCase {
+
+    func testNamesAndProperties() {
```

Comment:
Mostly a comment for future for the same situation:

I think this kind of test may not add a lot of value. It is similar to testing localization. The only way these would fail is if someone changed the strings. Then the goal would be that they're changed.

Reply:
I evaluated the value of these unit tests by considering this scenario: 

If an event name is mistakenly altered and merged into the trunk, how quickly can we detect and correct the error?

In my assessment, such an error would likely only be identified when someone searches for those events on Tracks, which doesn't happen frequently.

Additionally, events fired under an incorrect name cannot be fixed, causing irreversible damage.

Reply:
I think that scenario is unlikely. "Changing a string mistakenly" part. We can still test it sure. But I don't think a string would be changed unintentionally. After all, one cannot cause a regression but has to actually change the string itself. And then it should go unnoticed through code review. Not saying it is impossible, but I think it would be too unlikely that I wouldn't invest the time to write a test for it. But I wouldn't argue to remove an already written test for it. 

The same goes for serialization. We could unit test every entity / response type with the argument that somebody can change the CodingKey names unintentionally and cause the parsing to fail. However changing them can only be intenionally I think as one has to go to that specific string and change it.

Reply:
> I think that scenario is unlikely. "Changing a string mistakenly" part.

I agree with this, but another factor I took into consideration is Value vs Effort. The value of these unit tests is low, but so does the effort. If these unit tests were too challenging to add, then I wouldn't bother adding them.